invenio_rdm_records/resources/serializers/csl/__init__.py

```diff
@@ -5,9 +5,26 @@
 # Invenio-RDM-Records is free software; you can redistribute it and/or modify
 # it under the terms of the MIT License; see LICENSE file for more details.
 
-"""CSL JSON Serializers for Invenio RDM Records."""
+"""CSL JSON and  citation string serializers for Invenio RDM Records."""
 
+import re
+
+from citeproc import Citation, CitationItem, CitationStylesBibliography, \
+    CitationStylesStyle, formatter
+from citeproc.source.json import CiteProcJSON
+from flask import has_request_context, request
 from flask_resources.serializers import MarshmallowJSONSerializer
+from webargs import fields
+from webargs.flaskparser import FlaskParser
+from ...errors import StyleNotFoundRDMError
+
+try:
+    from citeproc_styles import get_style_filepath
```

Comment:
Why this try/except? I think it is in the `requirements` and it is not an extra.